src/main/java/seedu/jarvis/logic/parser/ParserUtil.java

```diff
@@ -108,6 +108,22 @@ public static Username parseUsername(String username) throws ParseException {
         return new Username(trimmedUsername);
     }
 
+    /**
+     * Parses an String score with integer values into a boolean.
```

Comment:
I think theres a typo here should be "Parses a String score with integer values into a boolean"

Reply:
Good catch! Updated.